content/security/rate-limiting.md

```diff
@@ -1,34 +1,79 @@
-### Rate limiting
+### Rate Limiting
 
-A common technique to protect applications from brute-force attacks is **rate-limiting**. Many Express packages exist to provide a rate-limiting feature. A popular one is [express-rate-limit](https://github.com/nfriedly/express-rate-limit).
+In this chapter we cover how to add rate limiting to your application in a more "NestJS Way" than using a package like `express-rate-limit`. To get started, you'll need to install the `@nestjs/throttler` package.
```

Comment:
A few notes:
1. I'd remove any references to "express" compatible packages
2. Since it's NestJS documentation (and `@nestjs/{..}` package), we should remove "in a more NestJS Way" as this is kinda obvious 😄 
3. Maybe we should leave the first sentence as it was just to let people know what this chapter is about (`A common technique to protect applications from brute-force attacks is **rate-limiting**.`)